cli/lib/env.js

```diff
@@ -1,7 +1,7 @@
 exports.get = get;
 exports.set = set;
 var def = {
-  "configFile": "rhmhcp.json"
+  "RHM_HCP_CFG": "rhmhcp.json"
```

Comment:
Could this be an env-var? Changed to match env var casing standards just in case